src/registrar/templates/includes/domains_table.html

```diff
@@ -1,10 +1,30 @@
 {% load static %}
 
 
-
 {% comment %} Stores the json endpoint in a url for easier access {% endcomment %}
 {% url 'get_domains_json' as url %}
+
+
+
 <span id="get_domains_json_url" class="display-none">{{url}}</span>
+
+<!-- Banner for when it's org model (org manager can view, domain manager can edit)  -->
```

Comment:
Condense this -- eg Org model expiration banner

Reply:
Open to feedback on this as well, but I'm a little strongly opinionated to leave this as is (open to different wording as long as it's clear on who can view/edit) just because it's a reminder to who can view versus edit as there are two different views per permissions access the user has 😬

Reply:
I'm good with keeping that distinction. Let's at least condense that beginning portion to Org model expiration banner for cleaner code